src/gui/search/searchwidget.cpp

```diff
@@ -129,7 +129,7 @@ void SearchWidget::fillCatCombobox()
     QList<QStrPair> tmpList;
     foreach (const QString &cat, m_searchEngine->supportedCategories())
         tmpList << qMakePair(SearchEngine::categoryFullName(cat), cat);
-    std::sort(tmpList.begin(), tmpList.end(), [](const QStrPair &l, const QStrPair &r) { return (l.first < r.first); } );
+    std::sort(tmpList.begin(), tmpList.end(), [](const QStrPair &l, const QStrPair &r) { return (QString::localeAwareCompare(l.first, r.first) < 0); } );
```

Comment:
>Here categories have only letters even when translated so I think operator < is fine here.

In Chinese, chars are sort by stroke numbers, `operator <` doesn't sort like this (instead it sort by unicode values).

>It is supposed to use the system locale for comparing, but what happens if the language chosen by the user(in qbt) is different than their locale? 

I would bet it uses app locale first.
I wonder why there is no trace of `QLocale::setDefault()` in qbt?
http://doc.qt.io/qt-4.8/qlocale.html#setDefault

>And I think locale benefits us when numbers or money amounts are involved(eg for decimal character). 

Last, in Qt doc: http://doc.qt.io/qt-4.8/qstring.html#localeAwareCompare
```
... Use this function to present sorted lists of strings to the user.
```